octoprint_costestimation/static/js/costestimation.js

```diff
@@ -14,15 +14,18 @@ $(function() {
         self.settings = parameters[1];
         self.loginState = parameters[2];
         self.filamentManager = parameters[3];
+        self.spoolManager = parameters[4];
 
         self.showEstimatedCost = ko.pureComputed(function() {
             return self.settings.settings.plugins.costestimation.requiresLogin() ?
                 self.loginState.isUser() : true;
         });
 
         self.showFilamentGroup = ko.pureComputed(function() {
-            return self.filamentManager === null || !self.settings.settings.plugins.costestimation.useFilamentManager();
-        })
+            var filamentManagerEnabled = self.filamentManager === null || !self.settings.settings.plugins.costestimation.useFilamentManager();
+            var spoolManagerEnabled = self.spoolManager === null || !self.settings.settings.plugins.costestimation.useSpoolManager();
+            return !filamentManagerEnabled && !spoolManagerEnabled;
```

Comment:
`filamentManagerEnabled` should be `filamentManagerDisabled`.

Reply:
Fix has been merged into PR, can you review again?